package.json

```diff
@@ -5,9 +5,14 @@
   "author": "illWatchers",
   "license": "GPL-3.0-or-later",
   "dependencies": {
-    "apollo-boost": "0.1.16",
+    "apollo-boost": "0.1.17",
+    "apollo-cache-inmemory": "1.3.6",
+    "apollo-link": "1.2.3",
+    "apollo-link-error": "1.1.1",
+    "apollo-link-http": "1.5.5",
```

Comment:
apollo-boost [already includes these packages](https://github.com/apollographql/apollo-client/blob/master/packages/apollo-boost/README.md). I assume that if we want to have more control over configuration let's don't include `apollo-boost` into our dependencies at all. [Link for further reading](https://www.apollographql.com/docs/react/advanced/boost-migration.html)

Reply:
@flashlee good catch, will adjust it 👍